hotham/src/systems/rendering.rs

```diff
@@ -397,7 +389,10 @@ mod tests {
         views: &Vec<openxr::View>,
     ) -> Result<(), String> {
         // Render the scene
-        let mut renderdoc = begin_renderdoc();
+
```

Comment:
I noticed that sometimes when running the test suite I would get a crash somewhere outside of Rust. I was thinking that it was something GPU related but after running the test suite in the debugger, it seems like there Renderdoc was crashing somewhere in its Vulkan Validation layers. It seems to only happen when running the entire test suite - not when running this test in isolation - so I've commented out the Renderdoc functionality for now.